File: APICuponSmart/src/java/modelo/pojo/respuesta/RespuestaTipoPromocion.java
package modelo.pojo.respuesta;

import java.util.List;
import modelo.pojo.entidad.TipoPromocion;

public class RespuestaTipoPromocion extends Mensaje{
    private List<TipoPromocion> contenido;
    
    public List<TipoPromocion> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<TipoPromocion> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaTipoPromocion.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.TipoPromocion

class RespuestaTipoPromocion(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<TipoPromocion>? = null
): Mensaje(error, mensaje)